### PR TITLE
VATRP-3778: Do not include Geolocation: header if we cannot get a location

### PR DESCRIPTION
**Pull request by ianblenke**

### CODE REVIEW

File: src/org/linphone/vtcsecure/LinphoneLocationManager.java [hunk 1]
```diff
@@ -31,36 +31,57 @@ public synchronized static LinphoneLocationManager instance(Activity a) {
...
-        return mLocationManager.isProviderEnabled(LocationManager.NETWORK_PROVIDER) || mLocationManager.isProviderEnabled(LocationManager.GPS_PROVIDER);
+        return mLocationManager.isProviderEnabled(LocationManager.PASSIVE_PROVIDER) || mLocationManager.isProviderEnabled(LocationManager.NETWORK_PROVIDER) || mLocationManager.isProviderEnabled(LocationManager.GPS_PROVIDER);
     }	
     
     
     public String userLocation() {
-    	if (!isLocationProviderEnabled()) return locationDenied;
-    	else if (userLocation == null || (userLocation.getLatitude()==0 && userLocation.getLongitude()==0)) return locationNotFound;
-    	else return "<geo:"+userLocation.getLatitude()+", "+userLocation.getLongitude()+">";
+    	if (!isLocationProviderEnabled()) return NULL; // locationDenied;
```
Comment (shareefalis) [inline src/org/linphone/vtcsecure/LinphoneLocationManager.java]: Use lower case null here

File: src/org/linphone/vtcsecure/LinphoneLocationManager.java [hunk 2]
```diff
@@ -31,36 +31,57 @@ public synchronized static LinphoneLocationManager instance(Activity a) {
...
+    	else if (userLocation == null || (userLocation.getLatitude()==0 && userLocation.getLongitude()==0)) return NULL; // locationNotFound;
```
Comment (shareefalis) [inline src/org/linphone/vtcsecure/LinphoneLocationManager.java]: use lower case null

File: src/org/linphone/CallManager.java [hunk 3]
```diff
@@ -64,7 +64,10 @@ public void inviteAddress(LinphoneAddress lAddress, boolean videoEnabled, boolea
 		String emergencyNumber_alt="+1"+emergencyNumber;
 
 		if ((lAddress.getUserName().startsWith(emergencyNumber)||lAddress.getUserName().startsWith(emergencyNumber_alt)) && LinphoneActivity.isInstanciated()) {
-			params.addCustomHeader("Geolocation", LinphoneLocationManager.instance(LinphoneActivity.instance()).userLocation());
+			String location = LinphoneLocationManager.instance(LinphoneActivity.instance()).userLocation();
+			if (location) {
```
Comment (shareefalis) [inline src/org/linphone/CallManager.java]: i think you mean if (location != null)

**Comment by patrickmwatson** [comment]
+1 LGTM

**Pull request merged**